### PR TITLE
Change message in workflow, update docs, pass namespace

### PR DESCRIPTION
**Pull request by ukff**

### CODE REVIEW

**Review by szwedm — APPROVED** [review]
/lgtm

File: docs/workflows.md [hunk 1]
```diff
@@ -7,4 +7,18 @@ The goal of the workflow is to register a module using the BTP Manager template
 If you want to control the workflow, you can set the following inputs in the GitHub UI:
 - Release tag - If not specified, the workflow takes a tag from the latest available release on GitHub and uses it. The tag can also be specified directly. The workflow validates if a given tag exists on any release and if it does, then uses it. Otherwise, the workflow breaks the execution.
 - Channel - There are three release channels to choose from: beta, fast, and regular. The options correspond to the subfolders in the module folder at the Kyma repository. 
-  
\ No newline at end of file
+
+
+---
+title: Auto update chart/resources
+---
```
Comment (IwonaLanger) [inline docs/workflows.md:14]:
```suggestion
## Auto-update chart/resources
```

File: docs/workflows.md [hunk 2]
```diff
@@ -7,4 +7,18 @@ The goal of the workflow is to register a module using the BTP Manager template
...
+
+The goal of workflow is to to update chart (module-chart/chart) to newest version, render resource templates from newest chart and create PR with changes. Workflow work on two shell scripts:
```
Comment (IwonaLanger) [inline docs/workflows.md:16]:
```suggestion
The goal of the workflow is to update the chart (module-chart/chart) to the newest version, render resource templates from the newest chart, and create a PR with the changes. The workflow works on two shell scripts:
```

File: docs/workflows.md [hunk 3]
```diff
@@ -7,4 +7,18 @@ The goal of the workflow is to register a module using the BTP Manager template
...
+
+- make-module-chart.sh - it download charts from (upstream)[https://github.com/SAP/sap-btp-service-operator] from realase tagged as 'latest' and place it in module-chart/chart.
```
Comment (IwonaLanger) [inline docs/workflows.md:18]:
```suggestion
- `make-module-chart.sh` - downloads the chart from the [upstream](https://github.com/SAP/sap-btp-service-operator) from the release tagged as 'latest' and places it in the `module-chart/chart`. 
```

File: docs/workflows.md [hunk 4]
```diff
@@ -7,4 +7,18 @@ The goal of the workflow is to register a module using the BTP Manager template
...
+
+---
+title: Auto update chart/resources
+---
+
+The goal of workflow is to to update chart (module-chart/chart) to newest version, render resource templates from newest chart and create PR with changes. Workflow work on two shell scripts:
+
+- make-module-chart.sh - it download charts from (upstream)[https://github.com/SAP/sap-btp-service-operator] from realase tagged as 'latest' and place it in module-chart/chart. 
+	
+- make-module-resources - it use helm to render kuberneter resources templates, as a base it takes chart from module-chart/chart and also takes values to override https://github.com/kyma-project/btp-manager/blob/main/module-chart/overrides.yaml. After helm templating finishes templating with appiled overrides, the generated resources are put into module-resources/apply, the resources which were in previous version but they are not in new version are placed under module-resource/delete
```
Comment (IwonaLanger) [inline docs/workflows.md:20]:
```suggestion
- `make-module-resources.sh` - uses Helm to render Kubernetes resources templates. As a base it takes a the chart from the `module-chart/chart` and also takes values to [override](https://github.com/kyma-project/btp-manager/blob/main/module-chart/overrides.yaml). After Helm finishes templating with the applied overrides, the generated resources are put into `module-resources/apply`. The resources that were used in the previous version but are not used in the new version are placed under `module-resource/delete`.
```

File: docs/workflows.md [hunk 5]
```diff
@@ -7,4 +7,18 @@ The goal of the workflow is to register a module using the BTP Manager template
...
+ 
+Both of above scripts are executed from the workflow but also can be triggered manually from developers computer. They are placed under hack/update/.
```
Comment (IwonaLanger) [inline docs/workflows.md:22]:
```suggestion
Both scripts are run from the workflow but can also be triggered manually from the developer's computer. They are placed under `hack/update/`.
```

File: docs/workflows.md [hunk 6]
```diff
@@ -7,4 +7,18 @@ The goal of the workflow is to register a module using the BTP Manager template
...
+
+We want to keep module-chart/chart in sync with (upstream)[https://github.com/SAP/sap-btp-service-operator], thus there should be not applied any manual changes there.
```
Comment (IwonaLanger) [inline docs/workflows.md:24]:
```suggestion
To keep `module-chart/chart` in sync with the [upstream](https://github.com/SAP/sap-btp-service-operator), you must not apply any manual changes there.
```

**Review by IwonaLanger — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**